bench/pytables_backend.py

```diff
@@ -65,9 +74,12 @@ def fill_table(self, con):
 
     def index_col(self, con, column, kind, optlevel, verbose):
         col = getattr(con.root.table.cols, column)
+        tmp_dir = os.path.join(self.datadir, "scratch2", "faltet")
```

Comment:
Provided that we are using a 'scratch2' in the datadir directory, it would be better to remote the trailing 'faltet'.